index.html

```diff
@@ -14,9 +15,9 @@
       const clientSideID = '';
 
       // Set flagKey to the feature flag key you want to evaluate
-      const flagKey = 'my-boolean-flag';
+      const flagKey = 'sample-feature';
 
-      // Set up the context properties. This context should appear on your
+      // Set up the evaluation context. This context should appear on your
       // LaunchDarkly contexts dashboard soon after you run the demo.
       const context = {
         kind: 'user',
```

Comment:
For line 33 below, I have a suggestion:

```jsx
div.replaceChild(document.createTextNode('Please edit index.html and set clientSideID to your LaunchDarkly client-side ID first. Please also set flagKey to your own flag key.'), div.firstChild);
```

Reply:
Given we have the comment on line 17, line 32 doesn't check the flag key is the default `sample-feature`, and the README.md also asks people to set the flag key if they have any, so I think keeping the current message is fine.